centre/centreFooter.jsp

```diff
@@ -235,9 +235,7 @@
             </c:if><!--
 
             --><li>
-              <a href="${changeTermsConditionsUrl}" data-track="click terms conditions" data-track-label="">
```

Comment:
This is good, however there is another terms and conditions you are missing in the file.  Remove that line as well